### PR TITLE
OSDOCS-8236: Reorganize SRE Access Page

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-8236](https://issues.redhat.com//browse/OSDOCS-8236): Reorganize SRE Access Page

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-8236

Link to docs preview:
https://67400--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-sre-access

QE review:
No QE needed as this was just a re-modularization and re-organization of content.

Additional information:

### CODE REVIEW

File: modules/sre-cluster-access.adoc [hunk 1]
```diff
@@ -10,20 +12,277 @@ SRE access to {product-title}
 ifdef::openshift-rosa[]
 (ROSA)
 endif::openshift-rosa[]
-clusters is controlled through several layers of required authentication, all of which are managed by strict company policy. All authentication attempts to access a cluster, as well as changes made within a cluster, are recorded within audit logs, along with the specific account identity of the SRE responsible for those actions. These audit logs help ensure that all changes made by SREs to a customer's cluster adhere to the strict policies and procedures that make up Red Hat's managed services guidelines.
+clusters is controlled through several layers of required authentication, all of which are managed by strict company policy. All authentication attempts to access a cluster and changes made within a cluster are recorded within audit logs, along with the specific account identity of the SRE responsible for those actions. These audit logs help ensure that all changes made by SREs to a customer's cluster adhere to the strict policies and procedures that make up Red Hat's managed services guidelines.
```
Comment (lpettyjo) [inline modules/sre-cluster-access.adoc:15]: "Clusters are controlled..."

File: modules/sre-cluster-access.adoc [hunk 2]
```diff
@@ -10,20 +12,277 @@ SRE access to {product-title}
...
 
-** SRE connects to the Red Hat VPN. The authentication to the VPN is completed by the Red Hat Corporate Identity and Access Management system (RH IAM). With RH IAM, SREs can be managed internally per organization via groups and existing on-boarding/off-boarding processes, and is multi-factor. After an SRE is authenticated and connected, the SRE can access the cloud services fleet management plane. Changes to the cloud services fleet management plane require many layers of approval and are maintained by strict company policy.
+** SRE connects to the Red Hat VPN. The authentication to the VPN is completed by the Red Hat Corporate Identity and Access Management system (RH IAM). With RH IAM, SREs are multifactor and can be managed internally per organization via groups and existing onboarding and offboarding processes. After an SRE is authenticated and connected, the SRE can access the cloud services fleet management plane. Changes to the cloud services fleet management plane require many layers of approval and are maintained by strict company policy.
```
Comment (lpettyjo) [inline modules/sre-cluster-access.adoc:21]: See IBM SG regarding "via". Change to "...per organization by groups..."

File: modules/sre-cluster-access.adoc [hunk 3]
```diff
@@ -10,20 +12,277 @@ SRE access to {product-title}
...
 
 ** With access granted to the fleet management plane, SRE uses various methods to access clusters, depending on network configuration.
 
 *** Accessing a private or public cluster: Request is sent through a specific Network Load Balancer (NLB) by using an encrypted HTTP connection on port 6443.
 
-*** Accessing a PrivateLink cluster: Request is sent to the Red Hat Transit Gateway, which then connects to a Red Hat VPC per region. The VPC that receives the request will be dependent on the target private cluster’s region. Within the VPC, there is a private subnet which contains the PrivateLink endpoint to the customer’s PrivateLink cluster.
+*** Accessing a PrivateLink cluster: Request is sent to the Red Hat Transit Gateway, which then connects to a Red Hat VPC per region. The VPC that receives the request will be dependent on the target private cluster's region. Within the VPC, there is a private subnet which contains the PrivateLink endpoint to the customer's PrivateLink cluster.
```
Comment (lpettyjo) [inline modules/sre-cluster-access.adoc:29]: Add comma. See IBM SG regarding "which/that": "...there is a private subnet, which contains the PrivateLink endpoint..."

File: modules/sre-cluster-access.adoc [hunk 4]
```diff
@@ -10,20 +12,277 @@ SRE access to {product-title}
...
 
+ifdef::openshift-rosa[]
+SREs access ROSA clusters through the web console or command-line interface (CLI) tools. Authentication requires multi-factor authentication (MFA) with industry-standard requirements for password complexity and account lockouts. SREs must authenticate as individuals to ensure auditability. All authentication attempts are logged to a Security Information and Event Management (SIEM) system.
```
Comment (lpettyjo) [inline modules/sre-cluster-access.adoc:32]: Remove hyphen. See RH Supplemental SG https://redhat-documentation.github.io/supplementary-style-guide/#plugin. "...command line interface (CLI)..."

File: modules/sre-cluster-access.adoc [hunk 5]
```diff
@@ -10,20 +12,277 @@ SRE access to {product-title}
...
+
+SREs generate a short-lived AWS access token for a reserved role using the AWS Security Token Service (STS). Access to the STS token is audit-logged and traceable back to individual users. Both STS and non-STS clusters use the AWS STS service for SRE access. For non-STS clusters, the `BYOCAdminAccess` role has the `AdministratorAccess` IAM policy attached, and this role is used for administration. For STS clusters, the `ManagedOpenShift-Support-Role` has the `ManagedOpenShift-Support-Access` policy attached, and this role is used for administration.
+
+[id="rosa-sre-sts-view-aws-account_{context}"]
+== SRE STS view of AWS accounts
+
+When SREs are on a VPN through two-factor authentication, they and Red Hat Support can assume the `ManagedOpenShift-Support-Role` in your AWS account. The `ManagedOpenShift-Support-Role` has all the permissions necessary for SREs to directly troubleshoot and manage AWS resources. Upon assumption of the `ManagedOpenShift-Support-Role`, SREs use a AWS Security Token Service (STS) to generate a unique, time-expiring URL to the customer's AWS web UI for their account. SREs can then perform multiple troubleshooting actions which include:
```
Comment (lpettyjo) [inline modules/sre-cluster-access.adoc:72]: "...multiple troubleshooting actions, which include:"

File: modules/sre-cluster-access.adoc [hunk 6]
```diff
@@ -10,20 +12,277 @@ SRE access to {product-title}
...
+--
+1. Limited to addressing common use cases such as failing deployments, upgrading a cluster, and replacing bad worker nodes.
+2. Red Hat associates have no access to customer data by default.
+3. SRE access to the AWS account is an emergency procedure for exceptional troubleshooting during a documented incident.
+4. Limited to what is granted through RBAC by the Customer Administrator and namespaces created by the user.
+--
+
+[id="rosa-policy-customer-access_{context}"]
+= Customer access
+Customer access is limited to namespaces created by the customer and permissions that are granted using RBAC by the Customer Administrator role. Access to the underlying infrastructure or product namespaces is generally not permitted without `cluster-admin` access. More information on customer access and authentication can be found in the "Understanding Authentication" section of the documentation.
```
Comment (lpettyjo) [inline modules/sre-cluster-access.adoc:192]: Change to: " For more information about customer access and authentication, see the "Understanding Authentication" section of the documentation.

File: modules/sre-cluster-access.adoc [hunk 7]
```diff
@@ -10,20 +12,277 @@ SRE access to {product-title}
...
+- To enable ROSA across your AWS organization, the customer is
+responsible for managing AWS Organizations administrators.
+
+- To enable ROSA across your AWS organization, the customer is
+responsible for distributing the ROSA entitlement grant using AWS License Manager.
+
+|Hardware and AWS global infrastructure
+|**AWS**
+
+- For information regarding physical access controls for AWS data centers, see link:https://aws.amazon.com/compliance/data-center/controls/[Our Controls] on the AWS Cloud Security page.
```
Comment (lpettyjo) [inline modules/sre-cluster-access.adoc:285]: Change to :" For information about physical access controls..."

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM!

File: modules/sre-cluster-access.adoc (hunk 1 above)
Comment (jneczypor) [reply modules/sre-cluster-access.adoc:15]: Going to leave this one "as is" because the full sentence is "SRE access to clusters is..."

**Review by jeana-redhat — COMMENTED** [review]
Content generally looks good! 

Since this is a reorganizing/modularizng effort, I think it's worth considering if this should all go in a single module. To me, it looks like a few different ones along these lines:

1. [SRE cluster access](https://67400--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-sre-access#sre-cluster-access_rosa-sre-access) `CONCEPT`
2. [Privileged access controls in ROSA](https://67400--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-sre-access#rosa-policy-privileged-access-control_rosa-sre-access) `REFERENCE`
3. [SRE access to AWS accounts](https://67400--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-sre-access#rosa-policy-sre-aws-infra-access_rosa-sre-access) `CONCEPT`
4. [SRE STS view of AWS accounts](https://67400--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-sre-access#rosa-sre-sts-view-aws-account_rosa-sre-access) `CONCEPT`
5. [SRE access through PrivateLink VPC endpoint service](https://67400--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-sre-access#rosa-sre-access-privatelink-vpc.adoc_rosa-sre-access) `CONCEPT`
6. [Red Hat support access](https://67400--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-sre-access#rosa-policy-rh-access_rosa-sre-access) `REFERENCE`
7. [Customer access](https://67400--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-sre-access#rosa-policy-customer-access_rosa-sre-access) `CONCEPT`
8. [Access approval and review](https://67400--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-sre-access#rosa-policy-access-approval_rosa-sre-access) `REFERENCE`

Those already have titles and IDs, so seems like the logical place to break things out.

Editing my comment to add: I could see the three small "SRE ..." sections (3-5 in the list above) as a single `CONCEPT` with three subheadings within it.

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**